IUBLibTech/ngao
# Bumps ArcLight from v0.3.3 to v0.4.0

Fixes  #101

# Summary 
This MR upgrades the ArcLight version where the fix for collapse and expand buttons has been implemented.

# Background
The collapse and expand buttons were not functioning properly in earlier versions of Microsoft Edge.  [This issue](https://github.com/projectblacklight/arclight/issues/1009) was addressed and resolved in [this MR](https://github.com/projectblacklight/arclight/pull/1011) and was included in the next release of ArcLight v0.4.0.  

# Related Issue
#101 

# Expected Behavior

Collapse and expand buttons function properly in Microsoft Edge version 44.

# Screenshots / Video
<details>
![image](https://user-images.githubusercontent.com/36549923/106829946-f5f03680-6641-11eb-8e2c-8c07d91346db.png)
</details>

# Dependencies

N/A

# Side Effects

Test suite indicates no adverse side effects of version upgrade.  Two facet fields were also added to the catalog controller but they are not visible to the user.

# Testing / Reproduction instructions

Step by step instructions on how to test this feature. Include links and username/passswords if necessary. Videos are especially helpful. Also, name any spec files that should be run for this feature, or to test any side effects from this feature.

1. Go to any single item's show page (in Browser: Microsoft Edge version 44 or earlier)
2. Scroll to Collection context
3. Expand and collapse content

# Deployment

Will be deployed to staging for QA.
